hecate.yaml

```diff
@@ -165,15 +165,24 @@ aws:
 ########################################
 # Argus - Webhooks Config
 ########################################
-argus:
-  # bucket is the partition name for storing webhooks.
+
+# migration describes how items should be pushed to Argus. 
+# (Optional) All the values below will be used as default 
```

Comment:
We should add this logic in the code meaning when owner and bucket values are not provided here, the code defaults them to `hecate-migrate` and `webhooks` respectively.